### ISSUE TITLE
incomplete documentation and inconsistencies

### ISSUE THREAD

**Issue by pmdci** [MicrosoftDocs/microsoft-365-docs#4026]
Script fails saying that you must connect to Exchange online first. Nowhere in the documentation it explains how to do this. It also says that all it needs is the PowerShell modules for Azure AD, nothing about Exchange.

I do have Exchange Module and I could do the connection before running the script. But then the problem is that it is not clear whether I should connect to source or the target Exchange.

Documentation should be updated to explain that Exchange Modules are required. Scripts should be update to connect to the correct Exchange.

In the documentation, Fabrikan often mentioned as the source and Contoso as the target. However in the scripts Fabrikan is the target and Contoso the source.

Down the line, a script also mentions a CSV file which the documentation never mentions how to create, nor it points to a documentation explaining how to create it.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c61c960c-f5c3-76f7-ac18-dfeaf30230e4
* Version Independent ID: 3dc607b2-869e-976f-80da-d73cfb1210e5
* Content: [Cross-tenant mailbox migration - Microsoft 365 Enterprise](https://docs.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-mailbox-migration?view=o365-worldwide)
* Content Source: [microsoft-365/enterprise/cross-tenant-mailbox-migration.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/cross-tenant-mailbox-migration.md)
* Product: **microsoft-365-enterprise**
* GitHub Login: @JoeDavies-MSFT
* Microsoft Alias: **josephd**

**Comment by msbemba** [comment]
@pmdci Thank you for your feedback.

1) In step-by-step-instructions-for-the-target-tenant-admin (step 3) we have discussed about connecting to Exchange online. - https://docs.microsoft.com/en-us/microsoft-365/enterprise/cross-tenant-mailbox-migration?view=o365-worldwide#step-by-step-instructions-for-the-target-tenant-admin
2) The scripts show fabrikam as the source and Contoso as a target.  Could you please elaborate on where you see this so that that can be fixed?
This is what I see - -ResourceTenantDomain fabrikam.onmicrosoft.com -TargetTenantDomain contoso.onmicrosoft.com
3)CSV is used for creating migration batches and details about getting the CSV file is provided in the FAQ.

Hope this helps!

Thanks
Sri

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and sharing this explanation. @pmdci Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by pmdci** [comment]
Come on lads... Do I need to do everything?

Here is its:

It says "Source tenant domain, such as fabrikam.onmicrosoft.com”.

[cid:image001.png@01D701F4.2327FC30]

Then at "The script will pause and ask you to accept or consent to the Exchange [...]" it says "SetupCrossTenantRelationshipForTargetTenant.ps1 -ResourceTenantDomain contoso.onmicrosoft.com".

[cid:image002.png@01D701F4.3C33BA70]


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by msbemba** [comment]
@pmdci Thank you for your feedback. I don't see the screenshots, unfortunately. But when you run the script (SetupCrossTenantRelationshipForTargetTenant.ps1) - There are some parameters and values that you need to mention (required) and one of them is -ResourceTenantDomain.

Hope this helps!

Thanks
Sri

**Comment by pmdci** [comment]
I added the text besides the screenshot as well.Everyone in my organisation sees the inconsistency. Fix it, don't fix it. I am done giving you guys feedback. On 15 Feb 2021 18:49, Sriraman M S <notifications@github.com> wrote:
@pmdci Thank you for your feedback. I don't see the screenshots, unfortunately. But when you run the script (SetupCrossTenantRelationshipForTargetTenant.ps1) - There are some parameters and values that you need to mention (required) and one of them is -ResourceTenantDomain.
Hope this helps!
Thanks
Sri

**Comment by pmdci** [comment]
You do understand I am NOT asking you for help and all I am doing is pointing an inconsistency on your documentation, that everyone) here at my end) sees but you? On 15 Feb 2021 18:56, Pedro Mac Dowell da Costa Innecco <pinnecco@outlook.com> wrote:I added the text besides the screenshot as well.Everyone in my organisation sees the inconsistency. Fix it, don't fix it. I am done giving you guys feedback. On 15 Feb 2021 18:49, Sriraman M S <notifications@github.com> wrote:
@pmdci Thank you for your feedback. I don't see the screenshots, unfortunately. But when you run the script (SetupCrossTenantRelationshipForTargetTenant.ps1) - There are some parameters and values that you need to mention (required) and one of them is -ResourceTenantDomain.
Hope this helps!
Thanks
Sri

**Comment by pmdci** [comment]
Here is the screenshot of the inconsistency in IMGUR: https://imgur.com/a/LcNgc4V
[https://i.imgur.com/ajl5F4O.png?fb]<https://imgur.com/a/LcNgc4V>
imgur.com<https://imgur.com/a/LcNgc4V>
Imgur: The magic of the Internet
imgur.com

________________________________
From: Pedro Mac Dowell da Costa Innecco on behalf of Pedro Mac Dowell da Costa Innecco <guybrush_yyz@yahoo.com>
Sent: 13 February 2021 10:40
To: MicrosoftDocs/microsoft-365-docs <reply@reply.github.com>; MicrosoftDocs/microsoft-365-docs <microsoft-365-docs@noreply.github.com>
Cc: Mention <mention@noreply.github.com>
Subject: RE: [MicrosoftDocs/microsoft-365-docs] incomplete documentation and inconsistencies (#4026)


Come on lads... Do I need to do everything?



Here is its:



It says "Source tenant domain, such as fabrikam.onmicrosoft.com”.



[cid:image001.png@01D701F4.2327FC30]



Then at "The script will pause and ask you to accept or consent to the Exchange [...]" it says "SetupCrossTenantRelationshipForTargetTenant.ps1 -ResourceTenantDomain contoso.onmicrosoft.com".



[cid:image002.png@01D701F4.3C33BA70]





Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by msbemba** [comment]
@pmdci Thank you for the feedback. I now found multiple instances where the target and the source are incorrect. I will update the document and let you know. Sorry again for the confusion as I was only looking at the downloaded script.

Thanks
Sri

**Comment by pmdci** [comment]
That is because you didn't read my original post. On 17 Feb 2021 22:28, Sriraman M S <notifications@github.com> wrote:
@pmdci Thank you for the feedback. I now found multiple instances where the target and the source are incorrect. I will update the document and let you know. Sorry again for the confusion as I was only looking at the downloaded script.
Thanks
Sri

**Comment by yogkumgit** [comment]
@msbemba Please let me know when you update this.

**Comment by msbemba** [comment]
@pmdci  and @yogkumgit  Fabrikam and Contoso were interchanged in a few places. According to most parts of the document source is Contoso and Target is Fabrikam. So we have made sure that It is consistent throughout the document. I have requested the author to review the same and once he reviews and approves the  PR#https://github.com/MicrosoftDocs/microsoft-365-docs/pull/4204 the document will be live.

Thanks
Sri
